Lighthouse-Tests.md

```diff
@@ -82,6 +82,15 @@ def export_url(url):
 - **Common question:** What to do if your changes cause the lighthouse accessibility tests to fail?
 - The lighthouse logs will provide you most of the information you need (To see the logs, click on the details button on the right)
 
+## Failing Lighthouse Puppeteer Script
+
+Lighthouse tests might fail because the puppeteer script failed to properly set up the web page to run ligthouse tests on. To determine if 
+this is the case, check the log and see whether the line "Puppeteer script failed" was printed.
```

Comment:
In the wiki, we don't break lines within paragraphs. This helps avoid large diffs from text re-wrapping after we just change a few words

Reply:
Understood - in the new commit, I removed these newlines within each of the paragraphs.